grr/server/grr_response_server/gui/api_integration_tests/metadata_test.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Lint as: python3
+# -*- encoding: utf-8 -*-
+"""Tests for API client and metadata-related API calls."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+import pkg_resources
+
+from absl import app
+
+from grr_response_server.gui import api_integration_test_lib
+from grr.test_lib import skip
+from grr.test_lib import test_lib
+
+
+class ApiClientLibMetadataTest(api_integration_test_lib.ApiIntegrationTest):
+  """Tests metadata-related part of GRR Python API client library."""
+
+  @skip.If(
+    "openapi-spec-validator" not in {p.key for p in pkg_resources.working_set},
+    "The openapi-spec-validator module used for validating the OpenAPI "
+    "specification is not installed."
+  )
+  def testOpenApiDescriptionValidity(self):
+    import openapi_spec_validator
+    open_api_dict = self.api.GetOpenApiDescription()
```

Comment:
+1 point for using `openapi` over `open_api` (inconsistency with `openapi_spec_validator).

Reply:
Agreed.